README.md

```
@@ -4,205 +4,164 @@
 
-[![npm version](https://img.shields.io/npm/v/@open-wa/wa-automate.svg?color=green)](https://www.npmjs.com/package/@open-wa/wa-automate)
-![node](https://img.shields.io/node/v/@open-wa/wa-automate)
-[![Downloads](https://img.shields.io/npm/dm/@open-wa/wa-automate.svg)](https://www.npmjs.com/package/@open-wa/wa-automate)
-<a href="https://discord.gg/dnpp72a"><img src="https://img.shields.io/discord/661438166758195211?color=blueviolet&label=discord&style=flat" /></a>
+[**Documentation**](https://openwa.dev) &nbsp;•&nbsp;
+[**Easy API**](#quick-start-easy-api) &nbsp;•&nbsp;
+[**SocketClient**](#simple-automation-socketclient) &nbsp;•&nbsp;
+[**Embedded Runtime**](#deep-integration-embedded-runtime) &nbsp;•&nbsp;
+[**AI Agents (MCP)**](#ai-agent-integration-mcp) &nbsp;•&nbsp;
+[**Plugins**](#plugins-and-integrations) &nbsp;•&nbsp;
+[**Community**](#support)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)